packages/coverage/README.md

```diff
@@ -0,0 +1,35 @@
+[![Betterer](https://raw.githubusercontent.com/phenomnomnominal/betterer/master/website/static/img/header.png)](https://phenomnomnominal.github.io/betterer/)
+
+# `@betterer/coverage`
+
+[![npm version](https://img.shields.io/npm/v/@betterer/coverage.svg)](https://www.npmjs.com/package/@betterer/coverage)
+
+Code coverage test for [**`Betterer`**](https://github.com/phenomnomnominal/betterer).
+
+## Usage
```

Comment:
If this could follow the format of the other README.md files in the built-in tests and link to a docusaurus documentation page that would be great!

Reply:
done